test/Array/array_qsortr.js

```diff
@@ -0,0 +1,73 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Test our custom qsort_r implementation on xplat.
```

Comment:
Good question. If there is a reason for enabling these tests on Windows, I can do that. (we particularly test custom`qsortr` here)